### ISSUE TITLE
Wither Queen has wrong name listed/spelled incorrectly

### ISSUE THREAD

**Issue by DiddyG** [GTNewHorizons/GT-New-Horizons-Modpack#18382]
### Your GTNH Discord Username

AdityaG13

### Your Pack Version

2.7.0

### Your Server

Private Server

### Java Version

Java 8

### Type of Server

None

### Your Expectation

The Wither Queen from NEI is listed as this: 
![image](https://github.com/user-attachments/assets/b45d6fa2-97a9-46f2-aba0-8eed381078d9)

However, inside the Mega Apiary, it is listed as this: 
![image](https://github.com/user-attachments/assets/808f7e3e-5462-4dcc-973c-916bc0126cf8)

### The Reality

The name inside the Mega Apiary is listed wrong/spelled incorrectly

### Your Proposal

I'm hoping that either the bee could be changed back to say "Wither Queen" or, if it stays as "sparkling," that it is spelled correctly and the bee is named Sparkling, or is written in the lore so as not to confuse players. 

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by 54M44R** [comment]
Cannot reproduce on:
- ForestryMC 4.11.13
- MagicBees 2.10.8-GTNH
- Binnie 2.6.24
- neiaddons 1.18.0
- GT5-Unofficial 5.09.52.456

<img width="305" height="175" alt="Image" src="https://github.com/user-attachments/assets/6f8a50b9-e9b1-4f9f-afe0-70e44898d61f" />
<br>
<img width="511" height="59" alt="Image" src="https://github.com/user-attachments/assets/cc27421f-643b-4c4b-826f-8a653ccec8f1" />

**Comment by AdityaVG13** [comment]
> Cannot reproduce on:
> - ForestryMC 4.11.13
> - MagicBees 2.10.8-GTNH
> - Binnie 2.6.24
> - neiaddons 1.18.0
> - GT5-Unofficial 5.09.52.456
> 
> <img width="305" height="175" alt="Image" src="https://github.com/user-attachments/assets/6f8a50b9-e9b1-4f9f-afe0-70e44898d61f" />
> <br>
> <img width="511" height="59" alt="Image" src="https://github.com/user-attachments/assets/cc27421f-643b-4c4b-826f-8a653ccec8f1" />

Yes, this was fixed a long time ago. You won't be able to reproduce it.
